lib/file-paths.sql

```diff
@@ -4,6 +4,6 @@ SELECT
 FROM
   questions AS q
 WHERE
-  q.course_id = $course_id
+  q.course_id = $course_id -- leave the same?
```

Comment:
Yep. This is used to read files for a specific question, which may include "template" questions in the same course.